pom.xml

```diff
@@ -24,11 +24,11 @@
 		<java.version>1.8</java.version>
 		<javadoc.opts>-Xdoclint:none</javadoc.opts>
 		<checkstyle.config.location>etc/checkstyle/checkstyle.xml</checkstyle.config.location>
-		<dataflow.version>1.7.0.BUILD-SNAPSHOT</dataflow.version>
-		<app.starters.support.version>1.3.3.RELEASE</app.starters.support.version>
-		<spring.web.version>4.3.4.RELEASE</spring.web.version>
+		<dataflow.version>2.5.0.BUILD-SNAPSHOT</dataflow.version>
```

Comment:
Since this data flow version is for the REST client, we need to make sure this works ok for the previous versions of SCDF as well.

Reply:
This is explicitly overridden. See `scripts/tests.sh`